src/nccl_ofi_net.c

```diff
@@ -1091,9 +1091,14 @@ static ncclResult_t ofi_init(ncclDebugLogger_t logFunction)
 	}
 
 	/*
-	 * RDMAV_FORK_SAFE environment variable makes the rdma-core
-	 * library fork-safe. This significantly increases cost of memory
-	 * registration when huge pages are enabled.
+	 * FI_EFA_FORK_SAFE environment variable tells Libfabric to enable
+	 * fork-safe support in legacy versions of the rdma-core library.
+	 * This significantly increases cost of memory registration when
+	 * huge pages are enabled. Libfabric checks if additional handling
```

Comment:
The huge page fork support is not only slow but also unsafe with libfabric, therefore when FI_EFA_FORK_SAFE is set, libfabric will not use huge page.